models/article-models.js

```diff
@@ -28,6 +28,22 @@ const fetchArticleByArticleID = (id) => {
     })
 }
 
+const fetchCommentsByArticleID = (article_id) => {
+    return db.query(`SELECT comment_id, votes, created_at, author, body
+                     FROM comments 
+                     WHERE article_id = $1`, [article_id])
+    .then(({ rows }) => {
+        const comments = rows; 
+        if (!comments[0]) {
```

Comment:
We need to first check if the article exists! If we make a request for an article id that is not present in the database, but is valid - we will still get an empty array, just like we would if there were no associated comments. Therefore we need to reject the promise if the article doesn't exist -> so by the time we get empty rows back, we know it is indeed because there are no associated comments.